convert2rhel/checks.py

```diff
@@ -75,77 +89,79 @@ def perform_pre_ponr_checks():
 
 
 def check_convert2rhel_latest():
-    """Make sure that we are running the latest version of convert2rhel"""
-    gpg_path = os.path.join(utils.DATA_DIR, "gpg-keys", "RPM-GPG-KEY-redhat-release")
-    ssl_cert_path = os.path.join(utils.DATA_DIR, "redhat-uep.pem")
-    repo_content = (
-        "[convert2rhel]\n"
-        "name=Convert2RHEL Repository\n"
-        "baseurl=https://cdn.redhat.com/content/public/convert2rhel/$releasever/x86_64/os/\n"
-        "gpgcheck=1\n"
-        "enabled=1\n"
-        "sslcacert=%s\n"
-        "gpgkey=file://%s\n" % (ssl_cert_path, gpg_path)
-    )
+    """Make sure that we are running the latest downstream version of convert2rhel"""
+    logger.task("Prepare: Checking if this is the latest version of convert2rhel")
+
+    if not system_info.has_internet_access:
+        logger.warning("Skipping the check because no internet connection has been detected.")
+        return
 
     cmd = [
         "repoquery",
         "--releasever=%s" % system_info.version.major,
     ]
 
-    repo_dir = tempfile.mkdtemp(prefix="convert2rhel_repo.", dir=utils.TMP_DIR)
-    utils.mkdir_p(repo_dir)
-    repo_path = os.path.join(repo_dir, "convert2rhel.repo")
-    store_content_to_file(filename=repo_path, content=repo_content)
-
-    cmd.append("--setopt=reposdir=%s" % repo_dir)
+    cmd.append(
+        "--repofrom='convert2rhel,https://cdn.redhat.com/content/public/convert2rhel/$releasever/$basearch/os/'convert2rhel"
+    )
     cmd.append("convert2rhel")
```

Comment:
I know it's a bit late to suggest this, but I would recommend grouping those `cmd` appends you have into just one… The reason to that is that you don't have any conditions to append then dynamically, they are all after each other.

Probably something like this: 

```python
    cmd = [
        "repoquery",
        "--releasever=%s" % system_info.version.major,
        "--repofrom='convert2rhel,https://cdn.redhat.com/content/public/convert2rhel/$releasever/$basearch/os/'convert2rhel",
        "convert2rhel"
    ]
```

Reply:
Your not late at all, I appreciate the comments and,I agree with this change. I will make the change and push it along with the rest of them